includes/functions.php

```diff
@@ -18,7 +18,7 @@ function send_webmention( $source, $target ) {
  *
  */
 function get_webmention_form_text( $post_id ) {
-	return apply_filters( 'webmention_form_text', __( 'Respond on your own site? Send me a <a href="http://indieweb.org/webmention">Webmention</a> by writing something on your website that links to this post and then enter your post URL below.', 'webmention' ), $post_id );
+	return apply_filters( 'webmention_form_text', __( 'Respond on your own site, link to this post and enter the URL below to link it below. Want to update or remove the mention? Just enter it again to update.(<a href="http://indieweb.org/webmention">Learn More</a>)', 'webmention' ), $post_id );
```

Comment:
Can you try to rephrase that? I wouldn't use `below` two times here: `enter the URL below to link it below`

Reply:
I wouldn't mind some brainstorming on this, maybe some input from @chrisaldrich and others. But the original focused on the plumbing(webmention), over the user benefit. And it needs to tell people they can delete or update with the form to cover some of the privacy implications.

Reply:
Tried another phrasing. Thoughts?